packages/dd-trace/src/plugin_manager.js

```diff
@@ -71,6 +72,9 @@ module.exports = class PluginManager {
     }
     if (!this._tracerConfig) return // TODO: don't wait for tracer to be initialized
 
+    // TODO: move this somewhere else?
```

Comment:
I think this still needs a new home

Reply:
this TODO is for later when this file gets refactored. Roch said it would happen sometime since there are lots of TODO's in this file